src/katgpucbf/xbgpu/bsend.py

```diff
@@ -174,7 +174,7 @@ def timestamp(self, value: int) -> None:
 
     @staticmethod
     def _inc_counters(
-        data_shape: tuple[int, int, int], beam_dtype: np.dtype, beam_names: Sequence[str], future: asyncio.Future
+        data_shape: tuple[int, int, int], data_type: np.dtype, output_names: Sequence[str], future: asyncio.Future
```

Comment:
I'd rather call it `data_dtype` (or just `dtype`): "dtype" has a specific meaning in numpy that's more specific than just "type".